### PR TITLE
translate to chinese

### PR DESCRIPTION
**Pull request by nusr**
This repo is cool，amazing。

### CODE REVIEW

File: README-zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+
+> 永远不必再考虑 React 状态管理了，仅仅 200 字节的状态管理终极解决方案。
+
+- **React Hooks** _用做你所有的状态管理。_
+- **~200 bytes** _min+gz._
+- **熟悉的 API** _仅仅使用了 React。_
+- **最小 API** _只需 5 分钟学习。_
+- **TypeScript 编写** _使你更容易编写 React 代码。_
+
+但是，最重要的问题是：这比 Redux 更好吗？ 答案是。
```
Comment (f48vj) [inline README-zh-cn.md]: Well 是比较婉转的，不是肯定式的

File: README-zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,465 @@
+# Unstated Next
+
+> 永远不必再考虑 React 状态管理了，仅仅 200 字节的状态管理终极解决方案。
+
+- **React Hooks** _用做你所有的状态管理。_
```
Comment (f48vj) [inline README-zh-cn.md]: them 指的是什么？没翻译

File: README-zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+- **~200 bytes** _min+gz._
+- **熟悉的 API** _仅仅使用了 React。_
```
Comment (f48vj) [inline README-zh-cn.md]: as intended ？

File: README-zh-cn.md [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+- **最小 API** _只需 5 分钟学习。_
+- **TypeScript 编写** _使你更容易编写 React 代码。_
```
Comment (f48vj) [inline README-zh-cn.md]: **type** your React code

File: README-zh-cn.md [hunk 5]
```diff
@@ -0,0 +1,465 @@
...
+-     <Counter.Provider value={counter}>
++     <Counter.Provider>
+        <CounterDisplay />
+        <CounterDisplay />
+      </Counter.Provider>
+    )
+  }
+```
+
+如果你正在使用 TypeScript（我鼓励你了解更多关于它的信息），这也有助于 TypeScript 的内置推断做得更好。 只要你编写自定义 hook ,其他所有内容都可以正常工作。
```
Comment (f48vj) [inline README-zh-cn.md]: 只要你编写自定义 hook ？ 这里的意思是如果你的自定义 hook 类型是完善的，那么类型都会自动推断。

**Comment by jamiebuilds** [comment]
Why was this closed and deleted?

**Comment by nusr** [comment]
Translation is not good.

**Comment by jamiebuilds** [comment]
That's okay, nothing is ever perfect, and others can come help. Like @f48vj was helping out. Try updating your PR and we can merge it

**Comment by f48vj** [comment]
Don't be afraid to make mistakes, we appreciate your contribution.

**Comment by nusr** [comment]
Ok.I will do it later.

**Pull request closed without merging**